streams/src/main/java/org/apache/kafka/streams/state/ReadOnlySessionStore.java

```diff
@@ -226,14 +323,14 @@
      * Get the value of key from a single session.
      *
      * @param key                    the key to fetch
-     * @param earliestSessionEndTime start timestamp of the session
-     * @param latestSessionStartTime end timestamp of the session
+     * @param sessionStartTime start timestamp of the session
+     * @param sessionEndTime end timestamp of the session
```

Comment:
I think we need to update the `@return` javadoc as `null if either no session with the exact start and end timestamp exist, or the specified key cannot be found in that session`, same below.